controllers/coral_placer.py

```diff
@@ -1,17 +1,33 @@
-from magicbot import StateMachine, timed_state
+import math
 
-from components.coral_placer import CoralPlacerComponent
+from magicbot import StateMachine, state, timed_state
+
+from components.algae_manipulator import AlgaeManipulatorComponent
+from components.wrist import WristComponent
 
 
 class CoralPlacer(StateMachine):
-    coral_placer_component: CoralPlacerComponent
+    wrist: WristComponent
+    algae_manipulator_component: AlgaeManipulatorComponent
+
+    WRIST_TILT_POINT = math.radians(-10.0)
 
     def __init__(self):
         pass
 
     def place(self):
         self.engage()
 
-    @timed_state(duration=2.0, first=True, must_finish=True)
+    @state(first=True, must_finish=True)
+    def lifting_wrist(self):
+        self.wrist.tilt_to(self.WRIST_TILT_POINT)
```

Comment:
You'll want to only do this once, otherwise you'll interrupt the trapezoid profile.